deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Product.wxs

```diff
@@ -127,6 +123,9 @@
                      Key="System\CurrentControlSet\Services\WAS">
           <RegistryValue Type="multiString" Name="Environment" Value="COR_ENABLE_PROFILING=1[~]COR_PROFILER=$(var.ProfilerCLSID)[~]DD_PROFILER_PROCESSES=w3wp.exe" Action="append"/>
         </RegistryKey>
+        <!-- W3SVC depends on WAS, so stopping WAS stops both services, and starting W3SVC starts both -->
```

Comment:
I think we might need a Condition here: http://wixtoolset.org/documentation/manual/v3/xsd/wix/condition.html

Reply:
i reverted this change for now, this PR only removes the reboot now